keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -282,102 +319,157 @@ not always required) or even code snippets. If there's any ambiguity
 about HOW your proposal will be implemented, this is the place to
 discuss them. -->
 
-We will expose a endpoint `/discovery` that will support JSON, and
-protobuf and gzip compression. The endpoint will serve the aggregated
-discovery document for all types that a Kubernetes cluster supports.
+The current discovery endpoints `/api` and `/apis` will accept a new
+content negotiation type `Accept:
+application/json;as=APIGroupDiscoveryList;v=v1beta1;g=meta.k8s.io` for
```

Comment:
for the non-http initiated, accept headers are normally additive in this way?

Reply:
yes

Reply:
yeah, we use this for requesting table formatted responses from the API server today (xref https://github.com/kubernetes/kubernetes/blob/8db0d1a8e02b2ce127c7f20b1a1ca22b0ce24742/staging/src/k8s.io/client-go/metadata/metadata.go#L186)